### DISCUSSION TITLE
Ошибка Installing podman and utilities

### DISCUSSION THREAD

**Discussion by punchedscull** [shvchk/padavan-builder#5] [General]
Здравствуйте, хорошее дело планируете. у меня ничего не получалось . потому что ubuntu  24.04 wsl2 . На 22.04 все запустилось. спасибо. А может подскажете когда в редактор конфига захожу, какие надо раскомментить строки, чтобы можно было vpn на роутере tl wr840n v5 заработал

**Comment by shvchk** [comment]
Ошибка `E: Could not get lock /var/lib/apt/lists/lock. It is held by process 23183 (apt-get)` означает, что в текущий момент происходит установка / обновление какого-либо ПО (в данном случае процессом `apt-get` с PID 23183). Параллельное выполнение нескольких таких системных операций запрещено, чтобы они не могли помешать друг другу и привести систему в нестабильное / неопределённое состояние. Поэтому скрипт не может установить необходимое ему ПО. Нужно сначала завершить уже начатый процесс установки / обновления ПО.

В интернете есть многочисленные статьи с более подробным разбором ситуации, их можно найти, проведя поиск по вышеуказанной ошибке :wink:
